docs/content/architecture/namespace.md

```diff
@@ -62,146 +60,302 @@ rules:
       - create
       - update
       - delete
+```
+
+### `readonly`
+
+While in this mode the PostgreSQL Operator is still able to listen for  namespace events within the
+Kubernetetes cluster, and then create and run and/or remove controllers as namespaces are added,
+updated and deleted.  Therefore, the Operator is able to dynamically respond to namespace events in 
+the Kubernetes cluster.  However, while in this mode the Operator is unable to create, delete or
+update namespaces itself, which means these actions must be performed externally by an
+administrator.
```

Comment:
In `readonly` mode, the PostgreSQL Operator is still able to listen to namespace events within a Kubernetes cluster, but it can no longer modify (create, update, delete) namespaces. For example, if a Kubernetes administrator creates a namespace, the PostgreSQL Operator can respond and add its controllers to that namespace.

Reply:
Updated per recommendation.